File: Work/NowCoder/Theonlineprogramming/RealQuestion/2019/Pro1035/Pro1035_1.c
/*************************************************************************
    > File Name: Pro1035_1.c
    > Author: lwei2
    > Mail: lwei2@IT.com 
    > The Description :
 ************************************************************************/

#include <stdio.h>

int main(void)
{
	int n,i;
	int sum = 0;
	scanf("%d",&n);
	if(n == 1)
		printf("%d",1);
	else
	{
		for(i = 0; sum < n; i++)
		{
			sum += i;
		}
		printf("%d",i-1);
	}
	return 0;
}

File: Work/NowCoder/Theonlineprogramming/RealQuestion/2019/Pro1035/Pro1035_1.java
/*************************************************************************
    > File Name: Pro1035_1.java
    > Author: lwei2
    > Mail: lwei2@IT.com 
    > The Description :
 ************************************************************************/
import java.util.*;

public class Main
{
	public static void main(String[] args)
	{
		Scanner input = new Scanner(System.in);
		int n = Integer.parseInt(input.nextLine());
		for(int i = 1; i <= n; i++)
		{
			if((1 + i)*i/2 >= n)
			{
				System.out.println(i);
				break;
			}
		}
	}
}
